codegen/realtime.py

```diff
@@ -260,10 +286,13 @@ def messages_to_lua(rtapi):
 	STATUS_MESSAGES = [ "StatusFollow", "StatusUnfollow", "StatusUpdate" ]
 	ALL_MESSAGES = CHANNEL_MESSAGES + MATCH_MESSAGES + MATCHMAKER_MESSAGES + PARTY_MESSAGES + STATUS_MESSAGES
 
+	NO_CALLBACK_MESSAGES = [ "MatchDataSend", "PartyDataSend" ]
```

Comment:
These two messages are the ones that seem to not set a `cid` in other client implementations (for instance nakama-dotnet)